ProductMatrix/pages/products.php

```diff
@@ -21,56 +21,59 @@
 ?>
 
 <br/>
-<table class="width50" align="center" cellspacing="1">
+<div class="table-container width50">
+	<table class="" align="center" cellspacing="1">
 
-<tr>
-<td class="form-title">Products</td>
-<td class="right" colspan="3"><?php if ( $t_can_manage ) print_bracket_link( plugin_page( 'config_page' ), plugin_lang_get( 'configuration' ) ); ?></td>
-</tr>
+	<tr>
+	<td class="form-title">Products</td>
+	<td class="right" colspan="3"><?php if ( $t_can_manage ) print_bracket_link( plugin_page( 'config_page' ), plugin_lang_get( 'configuration' ) ); ?></td>
+	</tr>
 
-<tr class="row-category">
-<td>Name</td>
-<td>Versions</td>
-<td>Platforms</td>
-<td>Actions</td>
-</tr>
+	<tr class="row-category">
+	<td>Name</td>
+	<td>Versions</td>
+	<td>Platforms</td>
+	<td>Actions</td>
+	</tr>
```

Comment:
I was refering to the user preferences screen. I fixed the layout. The PM preferences is good enough for us (looks as with 1.2)